src/basic/time-util.c

```diff
@@ -286,8 +286,7 @@ static char *format_timestamp_internal(
                 char *buf,
```

Comment:
maybe just rename this function to format_timestamp_style(), after all it is now identical to that

Reply:
done